lib/xsl/jats-to-html.xsl

```diff
@@ -1005,6 +1005,12 @@
         </span>
     </xsl:template>
 
+    <xsl:template match="named-content[@content-type='author-callout-style2']">
+        <span class="named-content author-callout-style2">
+            <xsl:apply-templates/>
+        </span>
+    </xsl:template>
+

```

Comment:
Found some more instances of named-content. So I think we need to have a generic approach to capture the others. Will investigate.